tests/rust/src/bin/interesting_paths.rs

```diff
@@ -16,7 +16,8 @@ unsafe fn test_interesting_paths(dir_fd: wasi::Fd, arg: &str) {
     assert_errno!(
         wasi::path_open(dir_fd, 0, "/dir/nested/file", 0, 0, 0, 0)
             .expect_err("opening a file with an absolute path"),
-        wasi::ERRNO_PERM
+        wasi::ERRNO_PERM,
+        wasi::ERRNO_NOTCAPABLE
```

Comment:
I think it's probably fine to have both for now, but I'd recommend opening an issue in the [WASI](https://github.com/WebAssembly/WASI/) repo to clarify that. Ideally this should be consistent.

Reply:
https://github.com/WebAssembly/WASI/issues/555